### DISCUSSION TITLE
`Keyboard.isHeld` returns `true` when the key is released with `Command` key held

### DISCUSSION THREAD

**Discussion by Ziphil** [excaliburjs/Excalibur#2607] [Q&A]
I found the following unexpected behaviour about `Keyboard.isHeld` on **MacOS Chrome**. Here is how to reproduce:

1. Press `A` key
2. Press `Command` key
3. Release `A` key
4. Release `Command` key

Then `engine.input.keyboard.isHeld(Input.Keys.A)` always returns `true` even though `A` key has been already released.

After a little investigation, I found others facing the same problem. The reason is that on **MacOS** (regardless of the browser), the `keyup` event does not fire when `Command` key is held. On Windows, everything is fine.

- https://stackoverflow.com/questions/73412298/keyup-event-not-firing-when-meta-key-is-held-mac-osx-only-how-to-handle-in
- https://github.com/electron/electron/issues/5188

The unexpected behaviour is thus not attributable to Excalibur, but to MacOS. However, as it is still unexpected, I suggest that some workaround be put on Excalibur. How about assuming that when `Control` key is released, the other keys are also released? How do you think?

**Comment by eonarheim** [comment]
Hi @Ziphil, thanks for bringing this up! I agree this is definitely unexpected.

I'll open an issue to implement a workaround 👍

**Reply by eonarheim** [reply]
https://github.com/excaliburjs/Excalibur/issues/2608

**Reply by eonarheim** [reply]
@Ziphil Fix merged into the latest main! https://github.com/excaliburjs/Excalibur/pull/2661
